lib/ASRouter.jsm

```diff
@@ -39,6 +39,7 @@ ChromeUtils.defineModuleGetter(this, "Sampling",
 const TRAILHEAD_CONFIG = {
   OVERRIDE_PREF: "trailhead.firstrun.branches",
   DID_SEE_ABOUT_WELCOME_PREF: "trailhead.firstrun.didSeeAboutWelcome",
+  INTERRUPT_EXPERIMENT_PREF: "trailhead.firstrun.interruptsExperiment",
```

Comment:
should this one be `INTERRUPTS_...` too? 🤔 #nit

Reply:
oh right!